readme.md

```diff
@@ -139,6 +139,9 @@ The easiest way to run it with Portainer on Linux is to use Portainer's stacks f
 ### How to run Nextcloud behind a Cloudflare Tunnel?
 Although it does not seems like it is the case but from AIO perspective a Cloudflare Tunnel works like a reverse proxy. So please follow the [reverse proxy documentation](./reverse-proxy.md) where is documented how to make it run behind a Cloudflare Tunnel.
 
+### Strato Linux VPS
+Stratos VPS are not recommended for running AIO, since they crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit, their Virtualization Software does not even fully support running docker
```

Comment:
As far as I know, does Virtuozzo not fully support docker, but we could remove this line since the main problem is the PID limit

Reply:
That was the case in the past, it's no longer like that. They support docker (unless you do something funky with the kernel like trying to load custom kernel modules and use them in docker).

I think removing this would make sense to clarify that also non AIO installations will fail due to the low PID limit.

Reply:
```suggestion
Stratos VPS are not recommended for running AIO, since they crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit.
```